arcade/types.py

```diff
@@ -419,6 +417,14 @@ def random(
 
         return cls(r, g, b, a)
 
+    def swizzle(self, swizzle_string: str) -> Tuple[int, ...]:
+        ret = []
+        for c in swizzle_string.lower():
+            if c not in 'rgba':
+                raise Exception("Swizzle string must only contain characters in [RGBArgba].")
```

Comment:
```suggestion
                raise ValueError("Swizzle string must only contain characters in [RGBArgba].")
```
I'd also consider outputting the offending value.

Reply:
Done the error improvement, tests to come!

Reply:
Thanks, @pushfoo, for the tests!